Gemfile.lock

```diff
@@ -341,6 +348,7 @@ GEM
     zeitwerk (2.6.6)
 
 PLATFORMS
+  ruby
```

Comment:
on which system is the platform ruby?

Reply:
don't know, was an unintentional change and i reverted it only

Reply:
see: https://bundler.io/man/gemfile.5.html#PLATFORMS